prime-dotnet-webapi/Migrations/20200427215637_SiteRegistrationUpdates.cs

```diff
@@ -0,0 +1,565 @@
+                nullable: true);
+
+            migrationBuilder.AlterColumn<int>(
+                name: "VendorId",
+                table: "Site",
+                nullable: true,
+                oldClrType: typeof(int),
+                oldType: "integer");
+
+            migrationBuilder.AlterColumn<int>(
+                name: "ProvisionerId",
+                table: "Site",
+                nullable: true,
+                oldClrType: typeof(int),
+                oldType: "integer");
+
+            migrationBuilder.AlterColumn<int>(
+                name: "LocationId",
+                table: "Site",
+                nullable: true,
+                oldClrType: typeof(int),
+                oldType: "integer");
+
+            migrationBuilder.AddColumn<DateTimeOffset>(
+                name: "SubmittedDate",
+                table: "Site",
+                nullable: true);
+
+            migrationBuilder.AddColumn<int>(
+                name: "PhysicalAddressId",
+                table: "Party",
+                nullable: true);
+
+            migrationBuilder.AlterColumn<int>(
+                name: "TechnicalSupportId",
+                table: "Location",
+                nullable: true,
+                oldClrType: typeof(int),
+                oldType: "integer");
+
+            migrationBuilder.AlterColumn<int>(
+                name: "PrivacyOfficerId",
+                table: "Location",
+                nullable: true,
+                oldClrType: typeof(int),
+                oldType: "integer");
+
+            migrationBuilder.AlterColumn<int>(
+                name: "OrganizationId",
+                table: "Location",
+                nullable: true,
+                oldClrType: typeof(int),
+                oldType: "integer");
+
+            migrationBuilder.AlterColumn<int>(
+                name: "AdministratorPharmaNetId",
+                table: "Location",
+                nullable: true,
+                oldClrType: typeof(int),
+                oldType: "integer");
+
+            migrationBuilder.AddColumn<int>(
+                name: "PhysicalAddressId",
+                table: "Location",
+                nullable: true);
+
+            migrationBuilder.AlterColumn<int>(
+                name: "EnrolleeId",
+                table: "BusinessEvent",
+                nullable: true,
+                oldClrType: typeof(int),
+                oldType: "integer");
+
+            migrationBuilder.AddColumn<int>(
+                name: "PartyId",
+                table: "BusinessEvent",
+                nullable: true);
+
+            migrationBuilder.AddColumn<int>(
+                name: "SiteId",
+                table: "BusinessEvent",
+                nullable: true);
+
+            migrationBuilder.InsertData(
+                table: "Vendor",
+                columns: new[] { "Id", "CreatedTimeStamp", "CreatedUserId", "Email", "Name", "UpdatedTimeStamp", "UpdatedUserId" },
+                values: new object[,]
```

Comment:
These are fine. I see lots of value in storing these email addresses (ie other integrations, automation, etc)